sjb/generated/test_pull_request_openshift_ansible_extended_conformance_install_crio.xml

```diff
@@ -596,8 +596,10 @@ cat &lt;&lt;SCRIPT &gt;&#34;${script}&#34;
 #!/bin/bash
 set -o errexit -o nounset -o pipefail -o xtrace
 cd &#34;\${GOPATH}/src/github.com/openshift/image-registry&#34;
-make build-images
-docker tag openshift/origin-docker-registry:latest &#34;openshift/origin-docker-registry:\$( cat /data/src/github.com/openshift/aos-cd-jobs/ORIGIN_COMMIT )&#34;
+if [[ &#34;\${PULL_BASE_REF}&#34; == &#34;master&#34; || &#34;\${PULL_BASE_REF}&#34; == &#34;release-3.7&#34; || &#34;\${PULL_BASE_REF}&#34; == &#34;release-3.8&#34; ]]; then
```

Comment:
@stevekuznetsov @jupierce not sure why my PR is generating changes to this file?  I pulled the latest master before I made my change.  The PR also just passed in Travis which I assume validates the generated files are correct?

Reply:
No, we don't have validation. Someone just forgot to generate :|